mkosi/installer/apt.py

```diff
@@ -292,6 +292,27 @@ def createrepo(cls, context: Context) -> None:
                 ),
             )  # fmt: skip
 
+        metadata = [
+            "Origin=mkosi",
+            f"Architectures={context.config.distribution.architecture(context.config.architecture)}",
+            "Description=mkosi local repository",
+        ]
+
+        with (context.repository / "Release").open("wb") as f:
+            run(
+                [
+                    "apt-ftparchive", "release", ".",
+                    *([f"-o=APT::FTPArchive::Release::{m}" for m in metadata]),
```

Comment:
Either alone will do already
```suggestion
                    *(f"-o=APT::FTPArchive::Release::{m}" for m in metadata),
```
or
```suggestion
                    *[f"-o=APT::FTPArchive::Release::{m}" for m in metadata],
```